systems/messaging/node-feeder/README.md

```diff
@@ -1,31 +1,31 @@
 # Node Feeder
 
-Device Feeder sends HTTP request to list of devices.
-
-
+Node Feeder sends HTTP request to list of nodes.
 
 ## Using Node Feeder
-To send request to a group of devices send a message to `amq.topic` exchange with routing key `request.cloud.node-feeder`.
+
+To send request to a group of Nodes send a message to `amq.topic` exchange with routing key `request.cloud.node-feeder`.
 The body should have the following format:
 
 ```
 {
```

Comment:
this should be proto-buffer, no more json

Reply:
addressed